.github/workflows/test.yml

```diff
@@ -22,4 +22,10 @@ jobs:
     - run: dotnet build -warnAsError -c Release
     - run: dotnet test -c Release /p:CollectCoverage=true /p:CoverletOutputFormat=opencover
     - uses: codecov/codecov-action@v1
-      if: ${{ matrix.os == 'macos-latest' }}
+      if: ${{ matrix.os == 'windows-latest' }}
+
+    - run: dotnet publish src\docfx\docfx.csproj -warnAsError -c Release -o $RUNNER_TEMP/publish
+    - uses: actions/upload-artifact@v2
+      with:
+        name: docfx-${{ matrix.os }}
+        path: $RUNNER_TEMP/publish
```

Comment:
Seems `$RUNNER_TEMP` was not expanded when this action was executed.

Reply:
Should it be `${{ env.RUNNER_TEMP }}` instead?

Reply:
Or perhaps `${{ runner.temp }}`, see [`runner` context](https://docs.github.com/en/actions/learn-github-actions/contexts#runner-context).